app/helpers/select_options_helper.rb

```diff
@@ -23,6 +23,16 @@ def select_provider_options(providers)
     ] + providers.map { |provider| OpenStruct.new(id: provider.id, name: "#{provider.name} (#{provider.code})") }
   end
 
+  def select_sort_options
+    [
+      [0, 'All time'],
+      [1, 'Last week'],
+      [2, 'Last month'],
+    ].map do |sort_option|
```

Comment:
Make the (frozen) array a constant?

Reply:
I've extracted to a var to make it clearer. Not so sure about extracting to a constant and putting into an initialiser right now? Maybe if it's used in another place perhaps?